.circleci/unittest/linux/scripts/install.sh

```diff
@@ -38,6 +38,7 @@ conda install -y -c "pytorch-${UPLOAD_CHANNEL}" pytorch ${cudatoolkit}
 
 # 2. Install torchaudio
 printf "* Installing torchaudio\n"
+git submodule update --init --recursive
```

Comment:
I'd prefer initializing and updating the submodules via the `git clone` command directly. Pointer on best way to do this?

Reply:
You need to provide custom cloning command in place of pre-defined `checkout` command in config file.
Though I do not see benefit of maintaining custom command over the maintaining cost if it's about preference.